### ISSUE TITLE
macOS CI is broken

### ISSUE THREAD

**Issue by jdm** [servo/mozjs#259]
```
$ if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then export PYTHON3="/usr/local/opt/python/bin/python3" && brew install "https://raw.githubusercontent.com/Homebrew/homebrew-core/3a877e3525d93cfeb076fc57579bdd589defc585/Formula/python@2.rb"; fi

Updating Homebrew...

==> Auto-updated Homebrew!

Updated 2 taps (homebrew/core and homebrew/cask).

==> Updated Formulae

Updated 15 formulae.

==> Updated Casks

Updated 1 cask.

Traceback (most recent call last):

	9: from /usr/local/Homebrew/Library/Homebrew/brew.rb:124:in `<main>'

	8: from /usr/local/Homebrew/Library/Homebrew/cmd/install.rb:132:in `install'

	7: from /usr/local/Homebrew/Library/Homebrew/cli/parser.rb:289:in `parse'

	6: from /usr/local/Homebrew/Library/Homebrew/cli/parser.rb:522:in `formulae'

	5: from /usr/local/Homebrew/Library/Homebrew/cli/parser.rb:522:in `map'

	4: from /usr/local/Homebrew/Library/Homebrew/cli/parser.rb:526:in `block in formulae'

	3: from /usr/local/Homebrew/Library/Homebrew/formulary.rb:351:in `factory'

	2: from /usr/local/Homebrew/Library/Homebrew/formulary.rb:138:in `get_formula'

	1: from /usr/local/Homebrew/Library/Homebrew/formulary.rb:142:in `klass'

/usr/local/Homebrew/Library/Homebrew/formulary.rb:227:in `load_file': Invalid usage: Installation of python@2 from a GitHub commit URL is unsupported! 'brew extract python@2' to stable tap on GitHub instead. (UsageError)

	12: from /usr/local/Homebrew/Library/Homebrew/brew.rb:155:in `<main>'

	11: from /usr/local/Homebrew/Library/Homebrew/brew.rb:157:in `rescue in <main>'

	10: from /usr/local/Homebrew/Library/Homebrew/help.rb:64:in `help'

	 9: from /usr/local/Homebrew/Library/Homebrew/help.rb:83:in `command_help'

	 8: from /usr/local/Homebrew/Library/Homebrew/help.rb:103:in `parser_help'

	 7: from /usr/local/Homebrew/Library/Homebrew/cli/parser.rb:289:in `parse'

	 6: from /usr/local/Homebrew/Library/Homebrew/cli/parser.rb:522:in `formulae'

	 5: from /usr/local/Homebrew/Library/Homebrew/cli/parser.rb:522:in `map'

	 4: from /usr/local/Homebrew/Library/Homebrew/cli/parser.rb:526:in `block in formulae'

	 3: from /usr/local/Homebrew/Library/Homebrew/formulary.rb:351:in `factory'

	 2: from /usr/local/Homebrew/Library/Homebrew/formulary.rb:138:in `get_formula'

	 1: from /usr/local/Homebrew/Library/Homebrew/formulary.rb:142:in `klass'

/usr/local/Homebrew/Library/Homebrew/formulary.rb:227:in `load_file': Invalid usage: Installation of python@2 from a GitHub commit URL is unsupported! 'brew extract python@2' to stable tap on GitHub instead. (UsageError)

The command "if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then export PYTHON3="/usr/local/opt/python/bin/python3" && brew install "https://raw.githubusercontent.com/Homebrew/homebrew-core/3a877e3525d93cfeb076fc57579bdd589defc585/Formula/python@2.rb"; fi" failed and exited with 1 during .
```

**Comment by jdm** [comment]
Trying to fix this in https://github.com/servo/mozjs/compare/master...jdm-patch-21?quick_pull=1.

**Comment by avr1** [comment]
@jdm Looks like a good fix, since macOS already comes with python 2.7
